.github/workflows/notify-pr-close.yml

```diff
@@ -0,0 +1,24 @@
+name: Notify on PR close
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  release-reminder:
+    name: Remind to release
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Comment on PR
+        uses: actions/github-script@v6
```

Comment:
is there a reason why you used this action instead of something simpler like https://github.com/marketplace/actions/comment-pull-request?

Reply:
The only reason for me was that [github-script](https://github.com/marketplace/actions/github-script) is marked verified on their marketplace, while the other one is not.

> Verified creator
> GitHub has verified that this action was created by https://github.com/actions.
> 
> [Learn more about verified Actions](https://docs.github.com/actions/automating-your-workflow-with-github-actions/using-github-marketplace-actions).

But github-script is a bit more generic than [Comment Pull Request](https://github.com/marketplace/actions/comment-pull-request). Let me know if you think I should use the latter.

Reply:
Irrelevant now since this is removed in https://github.com/canonical/rocks-toolbox/pull/2/commits/b284546e3b81a8032bd2d5e99bcc671ae914b6de.